dbptk-core/src/main/java/com/databasepreservation/modules/siard/out/metadata/TableIndexFileStrategy.java

```diff
@@ -16,17 +16,29 @@
 import com.databasepreservation.model.structure.ViewStructure;
 import com.databasepreservation.model.structure.type.Type;
 
-import dk.magenta.siarddk.tableindex.ColumnType;
-import dk.magenta.siarddk.tableindex.ColumnsType;
-import dk.magenta.siarddk.tableindex.ForeignKeyType;
-import dk.magenta.siarddk.tableindex.ForeignKeysType;
-import dk.magenta.siarddk.tableindex.PrimaryKeyType;
-import dk.magenta.siarddk.tableindex.ReferenceType;
-import dk.magenta.siarddk.tableindex.SiardDiark;
-import dk.magenta.siarddk.tableindex.TableType;
-import dk.magenta.siarddk.tableindex.TablesType;
-import dk.magenta.siarddk.tableindex.ViewType;
-import dk.magenta.siarddk.tableindex.ViewsType;
+import dk.sa.xmlns.diark._1_0.tableindex.ColumnType;
+import dk.sa.xmlns.diark._1_0.tableindex.ColumnsType;
+import dk.sa.xmlns.diark._1_0.tableindex.ForeignKeyType;
+import dk.sa.xmlns.diark._1_0.tableindex.ForeignKeysType;
+import dk.sa.xmlns.diark._1_0.tableindex.PrimaryKeyType;
+import dk.sa.xmlns.diark._1_0.tableindex.ReferenceType;
+import dk.sa.xmlns.diark._1_0.tableindex.SiardDiark;
+import dk.sa.xmlns.diark._1_0.tableindex.TableType;
+import dk.sa.xmlns.diark._1_0.tableindex.TablesType;
+import dk.sa.xmlns.diark._1_0.tableindex.ViewType;
+import dk.sa.xmlns.diark._1_0.tableindex.ViewsType;
+
+//import dk.magenta.siarddk.tableindex.ColumnType;
+//import dk.magenta.siarddk.tableindex.ColumnsType;
+//import dk.magenta.siarddk.tableindex.ForeignKeyType;
+//import dk.magenta.siarddk.tableindex.ForeignKeysType;
+//import dk.magenta.siarddk.tableindex.PrimaryKeyType;
+//import dk.magenta.siarddk.tableindex.ReferenceType;
+//import dk.magenta.siarddk.tableindex.SiardDiark;
+//import dk.magenta.siarddk.tableindex.TableType;
+//import dk.magenta.siarddk.tableindex.TablesType;
+//import dk.magenta.siarddk.tableindex.ViewType;
+//import dk.magenta.siarddk.tableindex.ViewsType;
```

Comment:
No, sorry - also forgot to remove those.

2015-10-12 10:29 GMT+02:00 Bruno Ferreira <notifications@github.com>:

> In
> dbptk-core/src/main/java/com/databasepreservation/modules/siard/out/metadata/TableIndexFileStrategy.java
> <https://github.com/keeps/db-preservation-toolkit/pull/74#discussion_r41733905>
> :
>
> > +import dk.sa.xmlns.diark._1_0.tableindex.TableType;
> > +import dk.sa.xmlns.diark._1_0.tableindex.TablesType;
> > +import dk.sa.xmlns.diark._1_0.tableindex.ViewType;
> > +import dk.sa.xmlns.diark._1_0.tableindex.ViewsType;
> > +
> > +//import dk.magenta.siarddk.tableindex.ColumnType;
> > +//import dk.magenta.siarddk.tableindex.ColumnsType;
> > +//import dk.magenta.siarddk.tableindex.ForeignKeyType;
> > +//import dk.magenta.siarddk.tableindex.ForeignKeysType;
> > +//import dk.magenta.siarddk.tableindex.PrimaryKeyType;
> > +//import dk.magenta.siarddk.tableindex.ReferenceType;
> > +//import dk.magenta.siarddk.tableindex.SiardDiark;
> > +//import dk.magenta.siarddk.tableindex.TableType;
> > +//import dk.magenta.siarddk.tableindex.TablesType;
> > +//import dk.magenta.siarddk.tableindex.ViewType;
> > +//import dk.magenta.siarddk.tableindex.ViewsType;
>
> is there any special reason to keep the comments?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keeps/db-preservation-toolkit/pull/74/files#r41733905>
> .
>



-- 
Magenta Aps

E-mail: andreas@magenta.dk
Tlf.: 81 77 16 98